core/src/main/res/layout/fragment_main.xml

```diff
@@ -65,5 +66,6 @@
     android:layout_height="match_parent"
     android:layout_gravity="end"
     android:layout_marginTop="24dp"
+    android:fitsSystemWindows="true"
```

Comment:
slightly confused by the amount of NavigationViews, do they all need to be here?